apps/crawler/src/subscribers/accounts-subscriber.service.ts

```diff
@@ -95,35 +160,46 @@ export class AccountsSubscriberService implements ISubscriberService {
     const log = {
       eventName,
       blockNumber,
-      rawAccountId: null as null | string,
-      accountId: null as null | string,
+      rawAddressValues: [],
+      processedAccounts: [],
     };
 
     try {
-      const rawAccountId = this.getAddressFromArgs(eventName, args);
+      const rawAddressValues = this.getAddressValues(eventName, args);
+      log.rawAddressValues = rawAddressValues;
 
-      if (!rawAccountId) {
-        throw new Error('Bad accountId');
+      if (!rawAddressValues.length) {
+        throw new Error('No addresses found');
```

Comment:
It will be looged later in catch statement:
`} catch (error) {
  this.logger.error({ ...log, error: error.message });
  this.sentry.instance().captureException({ ...log, error });
}`